### PR TITLE
Disable clang-format, clean up scripts

### PR DESCRIPTION
**Pull request by rubenwardy**
The configuration has been completely broken for a long time.

This PR disables clang-format from the CI

### CODE REVIEW

**Review by sfan5 — APPROVED** [review]
Looks good, concept wise agree too.

File: .github/workflows/cpp_lint.yml [hunk 1]
```diff
@@ -24,20 +24,21 @@ on:
...
-        run: |
-          sudo apt-get install clang-format-9 -qyy
 
-      - name: Run clang-format
-        run: |
-          source ./util/ci/lint.sh
-          perform_lint
-        env:
-          CLANG_FORMAT: clang-format-9
+#  clang_format:
```
Comment (lhofhansl) [inline .github/workflows/cpp_lint.yml:28]: Why not just remove it?

Comment (sfan5) [reply .github/workflows/cpp_lint.yml:28]: So it's easier to add back I assume, but `git revert` can also do that so it could as well be removed entirely.

**Review by SmallJoker — APPROVED** [review]

**Comment by nerzhul** [comment]
time to review all coding errors, arf :(

**Pull request merged**